### ISSUE TITLE
rseqc/geneBody_coverage2.py output not found 

### ISSUE THREAD

**Issue by whtns** [ewels/MultiQC#1072]
I see that this issue may be relevant https://github.com/ewels/MultiQC/issues/844

**Description of bug:**
Multiqc cannot recognize output from rseqc/geneBody_coverage2.py

**MultiQC Error log:**
```
[2019-12-03 11:27:05] multiqc                                            [DEBUG  ]  No MultiQC config found: /opt/miniconda3/envs/multiqc/lib/python3.6/site-packages/multiqc_config.yaml
[2019-12-03 11:27:05] multiqc                                            [DEBUG  ]  No MultiQC config found: /home/skevin/.multiqc_config.yaml
[2019-12-03 11:27:05] multiqc                                            [DEBUG  ]  No MultiQC config found: multiqc_config.yaml
[2019-12-03 11:27:05] multiqc                                            [DEBUG  ]  Command used: /opt/miniconda3/envs/multiqc/bin/multiqc . --verbose
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  Latest MultiQC version is v1.8
[2019-12-03 11:27:06] multiqc                                            [INFO   ]  This is MultiQC v1.8
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  Command     : /opt/miniconda3/envs/multiqc/bin/multiqc . --verbose
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  Working dir : /dataVolume/storage/single_cell_projects/sc_RB_devel/20180109-SHL-RDRB31-Hs_proj/output/rseqc_bw_output
[2019-12-03 11:27:06] multiqc                                            [INFO   ]  Template    : default
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  Running Python 3.6.7 | packaged by conda-forge | (default, Nov  6 2019, 16:19:42)  [GCC 7.3.0]
[2019-12-03 11:27:06] multiqc                                            [INFO   ]  Searching   : /dataVolume/storage/single_cell_projects/sc_RB_devel/20180109-SHL-RDRB31-Hs_proj/output/rseqc_bw_output
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  Analysing modules: custom_content, conpair, peddy, methylQA, phantompeakqualtools, qualimap, preseq, quast, qorts, rna_seqc, rsem, rseqc, busco, goleft_indexcov, disambiguate, supernova, deeptools, sargasso, verifybamid, mirtrace, happy, homer, macs2, theta2, snpeff, gatk, htseq, bcftools, featureCounts, fgbio, dedup, damageprofiler, biobambam2, mtnucratio, picard, prokka, samblaster, samtools, sexdeterrmine, bamtools, jellyfish, vcftools, longranger, stacks, bbmap, bismark, biscuit, hicexplorer, hicup, hicpro, salmon, kallisto, slamdunk, star, hisat2, tophat, bowtie2, bowtie1, kat, leehom, adapterRemoval, clipandmerge, cutadapt, flexbar, trimmomatic, skewer, sortmerna, biobloomtools, fastq_screen, afterqc, fastp, fastqc, minionqc, mosdepth, clusterflow, bcl2fastq, interop, flash, seqyclean
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  Using temporary directory for creating report: /tmp/tmpzxfgxpkl
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  Ignoring file as matched an ignore pattern: 2.geneBodyCoverage.pdf
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  Ignoring file as matched an ignore pattern: 1.geneBodyCoverage.pdf
[2019-12-03 11:27:06] multiqc.plots.bargraph                             [DEBUG  ]  Using matplotlib version 2.2.4
[2019-12-03 11:27:06] multiqc.plots.linegraph                            [DEBUG  ]  Using matplotlib version 2.2.4
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: custom_content
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: conpair
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: peddy
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: methylQA
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: phantompeakqualtools
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: qualimap
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: preseq
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: quast
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: qorts
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: rna_seqc
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: rsem
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: rseqc
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: busco
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: goleft_indexcov
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: disambiguate
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: supernova
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: deeptools
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: sargasso
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: verifybamid
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: mirtrace
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: happy
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: homer
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: macs2
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: theta2
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: snpeff
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: gatk
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: htseq
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: bcftools
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: featureCounts
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: fgbio
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: dedup
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: damageprofiler
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: biobambam2
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: mtnucratio
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: picard
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: prokka
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: samblaster
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: samtools
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: sexdeterrmine
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: bamtools
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: jellyfish
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: vcftools
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: longranger
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: stacks
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: bbmap
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: bismark
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: biscuit
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: hicexplorer
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: hicup
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: hicpro
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: salmon
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: kallisto
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: slamdunk
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: star
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: hisat2
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: tophat
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: bowtie2
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: bowtie1
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: kat
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: leehom
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: adapterRemoval
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: clipandmerge
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: cutadapt
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: flexbar
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: trimmomatic
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: skewer
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: sortmerna
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: biobloomtools
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: fastq_screen
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: afterqc
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: fastp
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: fastqc
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: minionqc
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: mosdepth
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: clusterflow
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: bcl2fastq
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: interop
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: flash
[2019-12-03 11:27:06] multiqc                                            [DEBUG  ]  No samples found: seqyclean
[2019-12-03 11:27:06] multiqc                                            [WARNING]  No analysis results found. Cleaning up..
[2019-12-03 11:27:06] multiqc                                            [INFO   ]  MultiQC complete


```

**File that triggers the error:**
Please drag and drop (and upload to the GitHub issue) an input file that I can use to replicate the error.

[sample1.geneBodyCoverage.txt](https://github.com/ewels/MultiQC/files/3918331/sample1.geneBodyCoverage.txt)
[sample2.geneBodyCoverage.txt](https://github.com/ewels/MultiQC/files/3918332/sample2.geneBodyCoverage.txt)

**MultiQC run details (please complete the following):**
 - Command used to run MultiQC: [eg. `multiqc .`]
multiqc .

 - MultiQC Version: [eg. MultiQC v1.1]
Multiqc v1.8

 - Operating System: [eg. macOS Sierra 10.12.5]
Ubuntu 18.04

 - Python Version: [eg. Python 2.7.12]
Python 3.6.7

 - Method of MultiQC installation: [eg. pip, conda, manual]
conda

 **Additional context**
I ran rseqc geneBody_coverag2.py on bigwig rather than running gene_BodyCoverage.py on bam
My search_patterns.yaml file contains the lines below, yet multiqc fails to find the attached files

```
rseqc/gene_body_coverage:
    fn: '*.geneBodyCoverage.txt'
```

**Comment by ewels** [comment]
Hi @whtns,

I'll close this issue as a duplicate, but thank you for the example data - I will use this when I get to working on the other issue.

Phil

**Comment by ewels** [comment]
Fixed in 2cae7a48aabf4f5e4fabc3da4a97ff139ab8a73e - see https://github.com/ewels/MultiQC/issues/844#issuecomment-636287274
